bootstrap/handlers/httpserver/httpserver.go

```diff
@@ -78,10 +78,11 @@ func (b *HttpServer) BootstrapHandler(
 
 	bootstrapConfig := container.ConfigurationFrom(dic.Get).GetBootstrap()
 
-	// Do not use Service.Host in the address.
-	// Using hostname it is not recommended because it will create a listener for at most one of the host's IP addresses
-	// which becomes an issue when using distributed orchestration.
-	addr := "0.0.0.0:" + strconv.Itoa(bootstrapConfig.Service.Port)
+	port := strconv.Itoa(bootstrapConfig.Service.Port)
+	addr := bootstrapConfig.Service.ServerBindAddr + ":" + port
+	if bootstrapConfig.Service.ServerBindAddr == "" {
```

Comment:
Add comment why defaulting to `Service.Host` when `Service.ServerBindAddr` is empty.

Reply:
Done!